etcdserver/api/v3rpc/rpctypes/error.go

```diff
@@ -113,6 +115,7 @@ var (
 	ErrPermissionDenied     = Error(ErrGRPCPermissionDenied)
 	ErrRoleNotGranted       = Error(ErrGRPCRoleNotGranted)
 	ErrPermissionNotGranted = Error(ErrGRPCPermissionNotGranted)
+	ErrRetryAuthRequired    = Error(ErrGRPCRetryAuthRequired)
```

Comment:
OK, I was thinking that where the retrying mechanism should be placed e.g. clientv3. But etcdserver would be a suitable place for the retrying.